src/main/scala/Validator.scala

```diff
@@ -60,6 +60,7 @@ class Validator(var tableCsvFile: URI, sourceUri: String = "") {
     }
     val table = schema.tables(tableUri.toString)
 
+    var allPrimaryKeyValues: Set[List[Any]] = Set()
```

Comment:
Would be a good idea to explain here why it has to be a `List` and can't be an `Array`.

Reply:
👍